### PR TITLE
Function to monitor any namespace

### PR DESCRIPTION
**Pull request by yashashreesuresh**
This PR solves: #17 
A single function named monitor_component() can be used to
monitor any namespace.

### CODE REVIEW

**Comment by Yashashree1997** [comment]
Please have a look @chaitanyaenr.

**Comment by rsevilla87** [comment]
Nice job @Yashashree1997 , just a quick suggestion. Why don't we switch the config file format to TOML or YAML?
We're using lists elements. So for the sake of simplicity it would be nice to use a config language that supports them natively

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+dist: xenial
```
[the remark this answers is not in the document]
Comment (Yashashree1997) [reply .travis.yml]: Yeah.

**Comment by Yashashree1997** [comment]
Thank you! @rsevilla87 
Yeah, using a config language that supports lists in this case would be better.

**Comment by chaitanyaenr** [comment]
@rsevilla87 Yaml is the way to go like you mentioned. Think, @mohit-sheth is working on https://github.com/openshift-scale/cerberus/issues/10 which will switch the tool to use yaml. @yashashreesuresh we can rebase the PR once it gets merged.

**Comment by mohit-sheth** [comment]
> @rsevilla87 Yaml is the way to go like you mentioned. Think, @mohit-sheth is working on #10 which will switch the tool to use yaml. @yashashreesuresh we can rebase the PR once it gets merged.

yup, will clean up things here and open a PR (hopefully today)

**Pull request closed without merging**